BIC_codes/FCS_estimate.py

```diff
@@ -0,0 +1,149 @@
+from functions.dFC_funcs import *
+import numpy as np
+import time
+import hdf5storage
+import scipy.io as sio
+import os
+os.environ["MKL_NUM_THREADS"] = '64'
+os.environ["NUMEXPR_NUM_THREADS"] = '64'
+os.environ["OMP_NUM_THREADS"] = '64'
+
+print('################################# CODE started running ... #################################')
+
+################################# Parameters #################################
+
+###### DATA PARAMETERS ######
+
+output_root = './../../../../../RESULTs/methods_implementation/'
```

Comment:
- consider putting paths in env variables or a config files rather than source files
- use pathlib to manipulate filesystem paths